### PR TITLE
docs: fix linter errors for security.md

### PR DESCRIPTION
**Pull request by dsnoeijer**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #47179

## What is the new behavior?
No more linter errors in the file.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/security.md [hunk 1]
```diff
@@ -27,21 +27,21 @@ For more information about how Google handles security issues, see [Google's sec
...
+| Don't alter your copy of Angular                                   | Private, customized versions of Angular tend to fall behind the current version and might not include important security fixes and enhancements. Instead, share your Angular improvements with the community and make a pull request.              |
 | Avoid Angular APIs marked in the documentation as "*Security Risk*" | For more information, see the [Trusting safe values](guide/security#bypass-security-apis) section of this page.                                                                                                                                    |
 
 </div>
 
 ## Preventing cross-site scripting (XSS)
 
 [Cross-site scripting (XSS)](https://en.wikipedia.org/wiki/Cross-site_scripting) enables attackers to inject malicious code into web pages.
-Such code can then, for example, steal user data \(in particular, login data\) or perform actions to impersonate the user.
+Such code can then, for example, steal user data, in particular login data, or perform actions to impersonate the user.
```
Comment (bob-watson) [inline aio/content/guide/security.md:39]:
```suggestion
Such code can then, for example, steal user and login data, or perform actions that impersonate the user.
```

File: aio/content/guide/security.md [hunk 2]
```diff
@@ -176,20 +176,20 @@ default-src 'self'; style-src 'self' 'unsafe-inline';
...
-As your project grows, however, you may need to expand your CSP settings beyond this minimum to accommodate additional features specific to your application.
+As your project grows, you may need to expand your CSP settings to accommodate extra features specific to your application.
 
 <a id="trusted-types"></a>
 
-### Enforcing Trusted Types
+### Enforcing trusted types
 
-We recommend the use of [Trusted Types](https://w3c.github.io/webappsec-trusted-types/dist/spec) as a way to help secure your applications from cross-site scripting attacks.
+It is recommended to use [Trusted Types](https://w3c.github.io/webappsec-trusted-types/dist/spec) as a way to help secure your applications from cross-site scripting attacks.
```
Comment (bob-watson) [inline aio/content/guide/security.md:185]:
```suggestion
It is recommended that you use [Trusted Types](https://w3c.github.io/webappsec-trusted-types/dist/spec) as a way to help secure your applications from cross-site scripting attacks.
```

File: aio/content/guide/security.md [hunk 3]
```diff
@@ -268,10 +270,12 @@ HTML constructed on the server is vulnerable to injection attacks.
 Injecting template code into an Angular application is the same as injecting executable code into the application:
 It gives the attacker full control over the application.
 To prevent this, use a templating language that automatically escapes values to prevent XSS vulnerabilities on the server.
-Don't generate Angular templates on the server side using a templating language; doing this carries a high risk of introducing template-injection vulnerabilities.
+Don't create Angular templates on the server side using a templating language. Doing this carries a high risk of introducing template-injection vulnerabilities.
```
Comment (bob-watson) [inline aio/content/guide/security.md:273]:
```suggestion
Don't create Angular templates on the server side using a templating language. This carries a high risk of introducing template-injection vulnerabilities.
```

**Review by bob-watson — CHANGES_REQUESTED** [review]
Thanks for the edits.
Just a few suggestions to tighten up the text.

**Comment by dsnoeijer** [comment]
@bob-watson 
Changes have been added.

**Review by bob-watson — APPROVED** [review]
LGTM

reviewed-for: global-docs-approvers

File: aio/content/guide/security.md [hunk 4]
```diff
@@ -176,20 +176,20 @@ default-src 'self'; style-src 'self' 'unsafe-inline';
 | `style-src 'self' 'unsafe-inline';` | Allows the page to load global styles from the same origin \(`'self'`\) and enables components to load their styles \(`'unsafe-inline'` - see [`angular/angular#6361`](https://github.com/angular/angular/issues/6361)\). |
 
 Angular itself requires only these settings to function correctly.
-As your project grows, however, you may need to expand your CSP settings beyond this minimum to accommodate additional features specific to your application.
+As your project grows, you may need to expand your CSP settings to accommodate extra features specific to your application.
 
 <a id="trusted-types"></a>
 
-### Enforcing Trusted Types
+### Enforcing trusted types
```
Comment (josephperrott) [inline aio/content/guide/security.md:183]: Since this is a proper noun, I believe we would still prefer to have "Enforcing Trusted Types"

**Review by josephperrott — APPROVED** [review]
LGTM

File: aio/content/guide/security.md (hunk 4 above)
Comment (dsnoeijer) [reply aio/content/guide/security.md]: Reverted it to Trusted Types.

**Review by alxhub — APPROVED** [review]
Reviewed-for: global-approvers

(globally approving as this is a docs-only cleanup and does not need `fw-security` review)

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit bbbd4c3ac87bf5ff0fb2b080cff4890d66ed0269.

**Pull request closed without merging**